docs/src/terminology.md

```diff
@@ -130,13 +134,15 @@ A [public key](terminology.md#public-key) and corresponding [private key](termin
 
 A fractional [native token](terminology.md#native-token) with the value of 0.000000001 [sol](terminology.md#sol).
 
+The smallest unit of the [native token](terminology.md#native-token), SOL. Equal to 0.000000001 [SOL](terminology.md#sol). Named after computer scientist [Leslie Lamport.](http://www.lamport.org/)
+
 ## leader
 
-The role of a [validator](terminology.md#validator) when it is appending [entries](terminology.md#entry) to the [ledger](terminology.md#ledger).
+Also known as the Proof of History Generator - The [validator](terminology.md#validator) that is selected to append [entries](terminology.md#entry) to the [ledger](terminology.md#ledger) for a given slot. To be accepted as valid, all entries need to be signed with the public key listed within the leader schedule for the slot being verified.
```

Comment:
The original definition looks better to me. This is just explaining what a "leader" is, not how it works.